src/App.jsx

```diff
@@ -14,6 +16,84 @@ export const goods = [
   'Garlic',
 ];
 
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+
+  function buttonSwitch(good) {
+    setSelectedGood(prev => (prev === good ? null : good));
+  }
+
+  function clearButton() {
+    setSelectedGood(null);
+  }
+
+  const isAnyGoodSelected = selectedGood !== null;
+
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
+
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={clearButton}
+          />
+        )}
+      </h1>
+      <table className="table">
+        <tbody>
+          {goods.map(good => {
+            let button;
+
+            if (isAnyGoodSelected && selectedGood === good) {
```

Comment:
will be something like that.
isAnyGoodSelected is redundant.
```suggestion
             selectedGood === good ? (
               <button
                  data-cy="RemoveButton"
                  type="button"
                  className="button is-info"
                  onClick={() => buttonSwitch(good)}
                >
                  -
                </button>
              ) : (
                 <button
                  data-cy="AddButton"
                  type="button"
                  className="button"
                  onClick={() => buttonSwitch(good)}
                >
                  +
                </button>
              )
              )
            
```